From 9bcb2ad6517eec8bf5188f6d59713a13b9ea5ace Mon Sep 17 00:00:00 2001
From: Beleswar Padhi <b-padhi@ti.com>
Date: Thu, 28 Nov 2024 15:31:13 +0530
Subject: [PATCH 1/3] arm: dts: am57xx-idk-common-u-boot: Include IPU Early
 Boot changes

The dra7-ipu-common-early-boot.dtsi defines the necessary nodes required
for Early Boot of IPUs like memory carveouts, page table carveouts, MMU
configuration and timers. Include this file in
am57xx-idk-common-u-boot.dtsi to enable Early Boot on all AM57XX IDK
devices.

Signed-off-by: Beleswar Padhi <b-padhi@ti.com>
---
 arch/arm/dts/am57xx-idk-common-u-boot.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/dts/am57xx-idk-common-u-boot.dtsi b/arch/arm/dts/am57xx-idk-common-u-boot.dtsi
index b07aea0048d..d0ce469f918 100644
--- a/arch/arm/dts/am57xx-idk-common-u-boot.dtsi
+++ b/arch/arm/dts/am57xx-idk-common-u-boot.dtsi
@@ -3,6 +3,7 @@
  * Copyright (C) 2019 Texas Instruments Incorporated - http://www.ti.com/
  */
 #include "omap5-u-boot.dtsi"
+#include "dra7-ipu-common-early-boot.dtsi"
 
 / {
 	xtal25mhz: xtal25mhz {
-- 
2.47.0

